src/modules/App/DesktopLayout.tsx

```diff
@@ -44,8 +44,8 @@ export const DesktopLayout: React.FC<DesktopLayoutProps> = (
           onProfileEditSuccess={onProfileEditSuccess}
           disableAutoSelect={disableAutoSelect}
           onChannelSelect={(channel) => {
-            setStartingPoint(null);
-            setHighlightedMessage(null);
+            setStartingPoint?.(null);
+            setHighlightedMessage?.(null);
```

Comment:
Cleander would be to make these two fns. not nullable
ie: `setStartingPoint(null); ` instead of `setStartingPoint?.(null); `

Reply:
similar for these setters than can be shared

Reply:
https://github.com/sendbird/sendbird-uikit-react/blob/feature/UIKIT-3476/Mobile-support-for-thread/src/modules/App/types.ts#LL30C3-L30C19